apps/sms/services/js/messaging/messaging_client.js

```diff
@@ -0,0 +1,54 @@
+/* global bridge */
+
+(function(exports) {
+  'use strict';
+
+  /**
+   * Name of the service that is responsible for messaging functionality.
+   * @type {string}
+   */
+  const SERVICE_NAME = 'messaging-service';
+  const TIMEOUT = 0;
```

Comment:
__optional nit:__ maybe comment here would be useful (why we need it and what 0 means)?